content/blog/2016/2016-04-14-replay-with-pipeline.adoc

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: "Replay a Pipeline with script edits"
+tags:
+- jenkins2
+- pipeline
+---
+
+[NOTE]
+====
+This is a cross-post of
+link:https://www.cloudbees.com/blog/replay-pipeline[an article] authored by
+Pipeline plugin maintainer link:https://github.com/jglick[Jesse Glick] on the
+link:https://www.cloudbees.com/blog/[CloudBees blog].
+====
+
+For those of you not checking their _Updates_ tab obsessively, Pipeline 1.14 [up
```

Comment:
*shrug* this is how @jglick originally worded it so I didn't care too much to modify it.